Update the Crier role

dellagustin opened the pull request:
This commit updates the Crier role with a link to the folder where the meeting notes are, plus an optional link to join the meeting.

To add the link to join the meeting, the property `links.meeting_link` has to be set in `_config.yml`.

Related issues:
- https://github.com/InnerSourceCommons/ispo-working-group/issues/87

![image](https://github.com/InnerSourceCommons/working-group-roles/assets/260113/130797ba-1167-445e-941a-43eb8ff925d6)

dellagustin committed:
Update the Crier role

This commit updates the Crier role with a link to the folder
where the meeting notes are, plus an optional link to join the
meeting.

To add the link to join the meeting, the property
`links.meeting_link` has to be set in `_config.yml`.

Related issues:
- https://github.com/InnerSourceCommons/ispo-working-group/issues/87

rrrutledge commented:
Awesome!  It looks like folder is already documented in config.yml - can you do the same for meeting_link?

dellagustin committed:
Update _config.yml

Added meeting_link to the template _config.yml file, follow up of c924fbe50b51bb7eec55d8672e84389106c0ff1c.

dellagustin-sap commented:
> Awesome! It looks like folder is already documented in config.yml - can you do the same for meeting_link?

Good point, done with ffe4a43ba6d88cc28b0303fc968ba45d7bbe5a1e.

rrrutledge reviewed (approved):
Great!  Thank you!

dellagustin-sap commented:
@rrrutledge could you please merge it for me? I don't have write access to this repository.
